cfme/tests/automate/test_customization_paginator.py

```diff
@@ -61,27 +61,26 @@ def test_paginator(some_dialogs, soft_assert, appliance):
         * During the cycling, assert the numbers displayed in the paginator make sense
         * During the cycling, assert the paginator does not get stuck.
     """
-    navigate_to(DialogCollection(appliance), 'All')
-    from cfme.web_ui import paginator
-    paginator.results_per_page(50)
-    paginator.results_per_page(5)
+    view = navigate_to(DialogCollection(appliance), 'All')
```

Comment:
**Question**
Why not `appliance.collections.service_dialogs`?

Reply:
@mshriver, Mike I don't want to change someone's tests.  I just add my enhancement.